packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -1240,7 +1261,11 @@ describe('lib/senders/email:', () => {
         });
       });
 
-      await mailer[type](MESSAGE);
+      const { updateTemplateValues } = opts;
+      const tmplVals = updateTemplateValues
+        ? updateTemplateValues(MESSAGE)
+        : MESSAGE;
```

Comment:
I added an optional way to re-shape the message/template values since a) the templates can contain logic based on these values, and b) we don't _actually_ pass the same set of values for all emails.